extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -497,13 +528,33 @@ private static String findKeystoreFileType(Path storePath) {
 
     private static HttpServerOptions createHttpServerOptions(HttpConfiguration httpConfiguration,
             LaunchMode launchMode, String websocketSubProtocols) {
+        if (!httpConfiguration.host.isPresent()) {
```

Comment:
cannot be `false`. It has a default value.
In addition, the caller need to be aware of the possibility to get `null`.

Reply:
It has a `null`-check, so it's ok, but it can still never be `false`.